js/contact-summary.js

```diff
@@ -0,0 +1,12 @@
+(function ($) {
+  $(document).ready(function () {
+    moveUserPictureNextToContactTitle();
+  });
+
+  /**
+   * Prepends the container image div on .crm-summary-contactname-block
```

Comment:
You are just restating the code. Tell me what this function does in practical terms, as I might have no idea what it means to move something in `.crm-summary-contactname-block`

Reply:
Got it. Let me update it